File: Checker source code with regex generated/Factorial check/test/tc3/tes3.pas
var a,b,d,i:integer;

begin
	// baca nilai a dan b
	readln(a,b); //comment wkwk
	// tulis nilai a + b
	writeln(a+b);
	{
		comment lagi
	}
	d := 1;
	for i:=2 to a do
	begin
		d := d*i;
	end;
	writeln(a,' ',d);
end.

File: Checker source code with regex generated/Factorial check/test/tc3/tes3.cpp
#include <cstdio>
#include <cstdlib>
#include <iostream>

int main(){
	int a,b;
	// baca nilai a dan b
	scanf("%d%d",&a,&b);//comment wkwk
	// tulis nilai a + b
	printf("%d\n",a+b);
	int d = 1;
	for (int i=2; i<=a; i++){
		d *= i;
	}
	printf("%d %d\n",a,d);
	return 0;
}
